pkg/gather/collect/collect_predicates.go

```diff
@@ -0,0 +1,10 @@
+package collect
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// IgnoreSecretsResourcePredicateFunc is a ResourcePredicateFunc that ignores Secret resources.
+func IgnoreSecretsResourcePredicateFunc(gv string, r *metav1.APIResource) bool {
+	return !(r.Kind == "Secret" && gv == "v1")
```

Comment:
Is there a reason why the ignored resources are not configurable by a flag?

I can see how this could easily be a StringSlice flag of `gv:k` strings.

For example:

`--exclude-types v1:Secret --exclude-types bitnami.com/v1alpha1:SealedSecret`

I have "invented" the syntax with `:` on the spot without much thought - there might be a better way.

Reply:
Sp the decision is to not go with my option 3, but to keep the secret and sealedsecret excluded unconditionally, right?

Completely fine with this.